project/AppDependencies.scala

```diff
@@ -2,14 +2,13 @@ import sbt._
 
 object AppDependencies {
 
-  private val customsApiCommonVersion = "1.57.0"
-
-  private val testScope = "test,it"
+  private lazy val customsApiCommonVersion = "1.57.0"
```

Comment:
Why do these need to be lazy? I don't remember seeing that pattern used anywhere before

Reply:
from my perspective it's a question of why not make it lazy? But don't think it really matters

Reply:
It doesn't need to be lazy, there is no benefit, and avoids the added overhead of synchronisation.